### ISSUE TITLE
Restore settings to default

### ISSUE THREAD

**Issue by justanotheranonymoususer** [Nordskog/SwiftKeyExi#7]
That's a great project, thanks!

Please add an option to reset the settings. That's especially relevant for the emoji page - before I understood all of the shortcuts, I've messed things around, and removed a couple of them. Now, it's not really possible to restore it to the original state, as you cannot reorder the emojis (can you?).

**Comment by Nordskog** [comment]
Yeah... the emoji tab apparently isn't terribly intuitive. 

Restoring defaults, along with backing up all settings is on the list of planned features,
in the meanwhile you can easily restore all the emoji panels to their original state.

The upper emoji panels are referred to as templates. When you first launch the module, the first 6? existing templates cannot be modified in any way. This is primarily so that I can update them with new emoji down the line. You can also add your own template panels, and either import from file or input emoji manually.

The bottom panels are what will actually appear inside of Swiftkey. These can be edited or deleted completely, with the exception of the "Recents" panel, which can only be moved. By default they are populated with the above templates.

If you want to restore the original panels, all you have to do is delete everything from the bottom row, and add each of the templates back in their place. New panel (Bottom Menu) -> Add all (Top menu) -> repeat.

https://gfycat.com/EmptyExcitableIberianbarbel

The general idea is that you import stuff into the template list, or use the existing panels there, then just add them in the order you want to an actual panel below.

**Comment by justanotheranonymoususer** [comment]
Thanks for the extensive explanation. I ended up not using the custom emojis pane, but I managed to screw up something else :)
In the Quick-Menu configuration, I accidentally created an additional option. I didn't find a way to remove it, and while trying, I created many more, having the maximum 10 in total. It would be great if I had a reset option there too, but for now, even an option to remove items would be great.

**Comment by Nordskog** [comment]
I haven't updated the main set of instructions in the while, but I do explain new features in the release post in the[ xda thread](https://forum.xda-developers.com/showpost.php?p=73993995&postcount=117)

tl;dr items in the quick-menu list can be swiped to the right to delete them.

**Comment by Nordskog** [comment]
Feature added in https://github.com/Nordskog/SwiftKeyExi/commit/1d304b04d4d52e0afee12ff07074aa04ec5ddae1
